daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/Node.scala

```diff
@@ -427,6 +427,11 @@ object Node {
   sealed trait WithTxValue3[F[+ _, + _, + _]] {
     type WithTxValue[+Nid, +Cid] = F[Nid, Cid, Transaction.Value[Cid]]
   }
+
+  final case class GlobalKeyWithMaintainers(
```

Comment:
Minor: it feels odd to see this in `Node.scala`, since it's not used here. I don't have a better suggestion, though.